tests/common/integration.rs

```diff
@@ -59,6 +67,11 @@ pub fn test_integration(package: impl AsRef<Path>, bindings: Option<String>) ->
     // We can do this since we know that wheels are built and returned in the
     // order they are in the build context
     for ((filename, supported_version), python_interpreter) in wheels.iter().zip(interpreter) {
+        if zig && build_context.target.is_linux() && !build_context.target.is_musl_target() {
+            assert!(filename
+                .to_string_lossy()
+                .ends_with("manylinux_2_12_x86_64.manylinux2010_x86_64.whl"))
```

Comment:
This assertion is wrong on aarch64 since it's minimal manylinux version is 2014.